shared/rust/src/domain/jig/module/body.rs

```diff
@@ -158,6 +158,9 @@ pub trait BodyExt<Mode: ModeExt, Step: StepExt>:
     /// will usually populate an inner .content
     fn new_mode(mode: Mode) -> Self;
 
+    /// Fetch the mode for this module
+    fn mode(&self) -> Option<Mode>;
```

Comment:
Previously there was no consistent way to fetch the mode, only to reset the mode.